core/src/main/kotlin/org/evomaster/core/output/service/PythonUtilsFileWriter.kt

```diff
@@ -0,0 +1,122 @@
+package org.evomaster.core.output.service
+
+import org.evomaster.core.EMConfig
+import org.evomaster.core.output.Lines
+import java.nio.file.Files
+import java.nio.file.Paths
+
+class PythonUtilsFileWriter {
+
+    private val utilsFilename = "em_test_utils.py"
+
+    fun writePythonUtilsFile(config: EMConfig) {
+        val lines = Lines(config.outputFormat)
+        addImports(lines)
+        writeResolveLocation(lines)
+        writeIsValidUriOrEmpty(lines)
+        writeExpectationHandler(lines)
+        saveToDisk(lines.toString(), config)
```

Comment:
this feels rather cumbersome.
I think a better approach would be to have the actual implementation of `em_test_utils.py`  under `core/src/main/resources`, so that it is part of generated jar file for core process.
then, you can read its content with something like: `Main::class.java.getResource("/em_test_utils.py").readText()`

Reply:
Sure! I'll change it to that